.eslintrc

```diff
@@ -1,6 +1,12 @@
 {
+  "env": {
+    "commonjs": true,
+    "es6": true
+  },
+  "extends": "eslint:recommended",
```

Comment:
So, I'm wondering if this would allow us to remove a some but not all of the stuff from lines 3 through 33.

Reply:
Yes - I'll go through them

Reply:
I could also go through them if you would prefer.

Reply:
No problem, already on it :)